FutureGateway/php/17_sayhello.data

```diff
@@ -0,0 +1,21 @@
+starting at: Mon Feb 13 13:07:26 UTC 2017
+running on: whoami: jobtest
+home: /home/jobtest
+listing home:
+total 0
+working directory: /home/jobtest/.jsaga/var/adaptor/ssh/84f44c83-c1fa-4a0e-819a-b35b66f99e18
+listing working directory:
+total 16
+-rw-rw-r-- 1 jobtest jobtest  64 Feb 13 13:07 sayhello.err
+-rw-rw-r-- 1 jobtest jobtest 232 Feb 13 13:07 sayhello.out
+-rw-rw-r-- 1 jobtest jobtest 672 Feb 13 13:07 sayhello.sh
+-rw-rw-r-- 1 jobtest jobtest 108 Feb 13 13:07 sayhello.txt
+User jobtest says: I am saying hello!
+Receiving sayhello.txt file
+!!!Say hello!!!
+---------------
+Demonstrative application for APIServer
+using I/O Sandboxing and arguments
+
+waiting for a while (2 minutes) ... done
+ending at: Mon Feb 13 13:09:26 UTC 2017
```

Comment:
I would suggest changing the suffix of this file from `.data` to a Markdown file.